eclair-core/src/main/scala/fr/acinq/eclair/io/PendingChannelsRateLimiter.scala

```diff
@@ -59,34 +64,59 @@ private class PendingChannelsRateLimiter(nodeParams: NodeParams, router: ActorRe
 
   import PendingChannelsRateLimiter._
 
-  private def restoring(channels: Map[PublicKey, Seq[PersistentChannelData]], pendingPublicNodeChannels: Map[PublicKey, Seq[ByteVector32]], pendingPrivateNodeChannels: Map[PublicKey, Seq[ByteVector32]]): Behavior[Command] = {
+  private def restoring(channels: Map[PublicKey, Seq[PersistentChannelData]], pendingPublicNodeChannels: Map[PublicKey, Seq[ByteVector32]], pendingPrivateNodeChannels: Map[PublicKey, Seq[ByteVector32]], cachedCommands: Seq[Command], sendGetNode: Boolean = true): Behavior[Command] = {
     channels.headOption match {
       case Some((remoteNodeId, pendingChannels)) =>
         val adapter = context.messageAdapter[Router.GetNodeResponse](r => WrappedGetNodeResponse(pendingChannels.head.channelId, r, None))
-        router ! GetNode(adapter, remoteNodeId)
+        if (sendGetNode) router ! GetNode(adapter, remoteNodeId)
         Behaviors.receiveMessagePartial[Command] {
           case AddOrRejectChannel(replyTo, _, _) =>
```

Comment:
It would be a bit dangerous, because that would block the `OpenChannelInterceptor` which would prevent us from opening channels where we are the initiator...I think it's simpler to reason about if we just reject those while we're initializing, our peer can always retry later.